README.md

```diff
@@ -108,6 +108,10 @@ Most important changes will be listed here, all other changes since `19.4.0` can
 - search for "NULL" in backend grids [#1203](https://github.com/OpenMage/magento-lts/pull/1203)
 - removed modules `Mage_Compiler`, `Mage_GoogleBase`, `Mage_Xmlconnect`, `Phoenix_Moneybookers`
 
+### Between OpenMage 19.4.18 / 20.0.15 and 19.4.19 / 20.0.16
```

Comment:
this is now wrong, but i think we should go for 19.5.0 and 20.1.0 or something like that

Reply:
Can we fix this in another PR?, when we made decission to change minor-release?

Reply:
yes but i'd at least use a number that doesn't correspond to a previous release, just no to confuse people

Reply:
For me it does not matter if it is called 19.4.123 or 19.5.0. I'll changed README.md when decession is made :)

Reply:
I understand and I agree, but 20.0.16 is already released, we can't say that in 20.0.16 php7.3 is needed since it's simply not true.

Reply:
Ah ... got it.

Reply:
Can you please make a "github suggestion" to this line? (currently workin on ... phpstan ...)